collection-spec/collection-spec.md

```diff
@@ -95,12 +96,12 @@ Summaries help to fully define Collections, especially if they don't link to any
 build tailored user interfaces for querying the data, by presenting the potential values that are available.
 Summaries should summarize all values in every Item underneath the collection, including in any nested sub-Catalogs.
 
-A summary for a field can be specified in two ways:
+A summary for a field can be specified in three ways:
 
 1. A set of all distinct values in an array: The set of values must contain at least one element and it is strongly recommended to list all values.
    If the field summarizes an array (e.g. [`instruments`](../item-spec/common-metadata.md#instrument)),
    the field's array elements of each Item must be merged to a single array with unique elements.
-2. Statistics in a [Stats Object](#stats-object): Statistics by default only specify the range (minimum and maximum values),
+2. A Range in a [Range Object](#range-object): Statistics by default only specify the range (minimum and maximum values),
    but can optionally be accompanied by additional statistical values.
    The range specified by the `minimum` and `maximum` properties can specify the potential range of values,
    but it is recommended to be as precise as possible.
```

Comment:
If I'm reading correctly, should have the 3rd way, describing the JSON Schema (and perhaps linking to the below paragraph on the JSON Schema summary)